make-release.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+# Based on https://github.com/che-incubator/chectl/blob/master/make-release.sh
+
+set -e
+set -u
+
+usage ()
+{   echo "Usage: ./make-release.sh <schema-version> <k8s-api-version>"
+    exit
+}
+
+if [[ $# -lt 2 ]]; then usage; fi
+
+VERSION=$1
+API_VERSION=$2
+
+if ! command -v hub > /dev/null; then
+  echo "[ERROR] The hub CLI needs to be installed. See https://github.com/github/hub/releases"
+  exit
+fi
+if [[ -z "${GITHUB_TOKEN}" ]]; then
```

Comment:
I checked the existing doc in https://github.com/devfile/api/blob/master/docs/proposals/versioning-and-release.md, we didn't update the instructions to set the GITHUB_TOKEN as part of the doc. Should we update it with that info so that anyone who needs to create a release in the future can just follow the instructions without hiccups?

Reply:
Good point, updated the design doc 👍